Danki2/Assets/Scripts/Abilities/MouseGamePositionFinder.cs

```diff
@@ -32,4 +33,16 @@ public Vector3 GetMouseGamePosition()
             return default;
         }
     }
+
+    /// <summary>
+    /// Returns the position of the mouse when projected onto the horizontal plane, of the
+    /// height set on this component, and then casts down to the floor.
+    /// </summary>
+    /// <returns></returns>
+    public Vector3 GetMouseFlooredMouseGamePosition()
```

Comment:
Should this be GetFlooredMouseGamePosition() ?
Otherwise the name isn't the easiest to interpret.